circle.yml

```
@@ -1488,6 +1502,16 @@ jobs:
           browser: firefox
           command: "npm run cypress:run"
 
+  "test-binary-against-cypress-realworld-app":
+    <<: *defaults
+    steps:
+      - test-binary-against-repo:
+          repo: cypress-realworld-app
+          browser: chrome
+          start-script: "start:ci"
```

in that case, probably better parameter name would be `server-start-command`
```suggestion
          server-start-command: "npm run start:ci"
```